setup.php

```diff
@@ -128,8 +128,6 @@ function quicktree_config_arrays() {
 
 	quicktree_check_upgrade();
 
-	api_plugin_register_realm('quicktree', 'quicktree.php', __('QuickTree Tree Management', 'quicktree'), 1);
```

Comment:
Is this registered elsewhere?

Reply:
That's a good point.  Let me see about that.